view/components/mainView.ts

```diff
@@ -0,0 +1,114 @@
+export function isValidEmail(email: string): string | true {
+    if (!email.includes('@')) {
+        return 'Uwzględnij znak "@" w adresie e-mail.';
+    }
+    if (!email.includes('.')) {
+        return 'Uwzględnij znak "." w adresie e-mail.';
+    }
+    return true;
+}
+
+export function isValidPhone(phone: string): boolean {
+    const phoneRegex = /^\d{9}$/;
+    return phoneRegex.test(phone);
+}
+
+export function mainView(): string {
+    return `
+        <html>
```

Comment:
Lepiej by było napisać to w osobnym pliku .html i czytać go przy odpalaniu serwera. To jest tylko mockup ale mógłby mieć podkreśloną składnię i IntelliSense.